src/app/booking/Intro.js

```diff
@@ -0,0 +1,95 @@
+"use client";
+
+import { Box, Container } from "@mui/material";
+import Typography from "@mui/material/Typography";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { useTheme } from "@mui/material/styles";
+import Grid from "@mui/material/Grid";
+import DemoStepper from "@/app/(homepage)/DemoStepper";
+
+function IntroText(props) {
+
+  const STEPS = [
+    "Select activities",
+    "Activities are added to your child’s calendar",
+    "Pay for all bookings in one go",
+  ];
+
+  return (
+    <Box textAlign="center" {...props}>
+      <Typography my={3} variant="h1">
+        Discover, book and track activities without clicking,
+        typing and scrolling marathons
+      </Typography>
+      <Typography
+        mt={5}
+        color="text.secondary"
+        variant="introSubheading"
+        display={{ xs: "none", md: "block" }}
+      >
+        We know when kids are going to be Out Of School so why not plan by
+        calendar instead by activities. It’s that simple, easy and intuitive.
+      </Typography>
+      <Typography
```

Comment:
Should be 19 px at desktop

Reply:
Why?
![image](https://github.com/digital-brother/ooscca-fe/assets/49399157/3bc3d174-3760-4f7b-baa3-1005de5d5468)